xpanse/api/asset_management/assets_management_v1.py

```diff
@@ -4,6 +4,7 @@
 from xpanse.endpoint import XpanseEndpoint
 from xpanse.iterator import XpanseResultIterator
 from xpanse.response import XpanseResponse
+from xpanse.types import RequestData, Filter
```

Comment:
Got it - nit, but can we name it to something like assets_management_utils_v1 or just assets_management_utils (maybe under a utils folder)? otherwise, lgtm

Reply:
I went with `assets_management_base` since it's a base class for the specific data type endpoints, thanks!